src/elasticSearchService.ts

```diff
@@ -388,10 +401,11 @@ export class ElasticSearchService implements Search {
     }
 
     // eslint-disable-next-line class-methods-use-this
-    private createURL(host: string, query: any, resourceType?: string) {
+    private createURL(host: string, query: any, tenantId: string, resourceType?: string) {
+        const tenantUrlFragment = tenantId ? `/tenant/${tenantId}` : '';
```

Comment:
use consistent naming from above
```suggestion
        const tenantPath = tenantId ? `/tenant/${tenantId}` : '';
```